src/cbc_sdk/platform/alerts.py

```diff
@@ -31,121 +31,136 @@
-    "threat_indicators": "ttps",
-    "watchlists": "watchlists.id",
-    "workflow.last_update_time": "workflow.change_timestamp",
-    "workflow.comment": "workflow.note",
-    "workflow.remediation": "workflow.closure_reason",
-    "workflow.state": "workflow.status",
-    "workload_kind": "k8s_kind",
-    "workload_name": "k8s_workload_name"
-}
-
-REMAPPED_WORKFLOWS_V6 = {
-    "workflow.last_update_time": "workflow.change_timestamp",
-    "workflow.comment": "workflow.note",
-    "workflow.remediation": "workflow.closure_reason",
-    "workflow.state": "workflow.status",
-}
-
-REMAPPED_NOTES_V6 = {
-    "last_update_time": "",
-
-}
-
-ALERTS_V6_MAPPINGS = {
-    "ml_classification_final_verdict": "alert_classification.classification",
-    "ml_classification_global_prevalence": "alert_classification.global_prevalence",
-    "ml_classification_org_prevalence": "alert_classification.org_prevalence",
-    "determination_value": "alert_classification.user_feedback",
-    "k8s_cluster": "cluster_name",
-    "backend_timestamp": "create_time",
-    "first_event_timestamp": "first_event_time",
-    "last_event_timestamp": "last_event_time",
-    "backend_update_timestamp": "last_update_time",
-    "k8s_namespace": "namespace",
-    "alert_notes_present": "notes_present",
-    "device_policy_id": "policy_id",
-    "device_policy": "policy_name",
-    "netconn_local_port": "port",
-    "netconn_protocol": "protocol",
-    "netconn_remote_domain": "remote_domain",
-    "netconn_remote_ip": "remote_ip",
-    "remote_k8s_namespace": "remote_namespace",
-    "remote_k8s_pod_name": "remote_replica_id",
-    "remote_k8s_kind": "remote_workload_kind",
-    "remote_k8s_workload_name": "remote_workload_name",
-    "k8s_pod_name": "replica_id",
-    "rule_id ": "rule_id",
-    "run_state": "run_state",
-    "device_target_value": "target_value",
-    "process_issuer": "threat_cause_actor_certificate_authority",
-    "process_name": "threat_cause_actor_name",
-    "process_publisher": "threat_cause_actor_publisher",
-    "process_sha256": "threat_cause_actor_sha256",
-    "primary_event_id": "threat_cause_cause_event_id",
-    "process_md5": "threat_cause_md5",
-    "parent_guid": "threat_cause_parent_guid",
-    "process_reputation": "threat_cause_reputation",
-    "ttps": "threat_indicators",
-    "watchlists.id": "watchlists",
-    "workflow.change_timestamp": "workflow.last_update_time",
-    "workflow.note": "workflow.comment",
-    "workflow.closure_reason": "workflow.remediation",
-    "workflow.status": "workflow.state",
-    "k8s_kind": "workload_kind",
-    "k8s_workload_name": "workload_name"
-}
-
-REMAPPED_WORKFLOWS_V7_TO_V6 = {
-    "change_timestamp": "last_update_time",
-    "note": "comment",
-    "closure_reason": "remediation",
-    "status": "state"
-}
-
-REMAPPED_NOTES_V7_TO_V6 = {
-    "last_update_time": "",
-
-}
+
+
+
 
 
 class Alert(PlatformModel):
     """Represents a basic alert."""
+    REMAPPED_ALERTS_V6 = {
```

Comment:
Why are two enumerations of mappings required?  REMAPPED_ALERTS_V6 and also ALERTS_V6_Mapping?

Reply:
REMAPPED_ALERTS_V6 is the v7 to v6 mapping then ALERTS_V6_Mapping is v6 to v7. This could be named better to make that more clear

Reply:
Incorprated in CPAPI_4813_V6_Unit_Tests